package.json

```diff
@@ -149,6 +150,7 @@
     "Matthew Piercey <piercey.matthew@gmail.com>",
     "Neale Pickett <neale@woozle.org>",
     "Robert <robert@n5qm.com>",
+    "Soura Dutta <duttasoura@gmail.com>"
```

Comment:
You appear to be missing a comma

Reply:
Sure, I'll fix that.

Reply:
It's done, the comma is there now. Sorry got a little busy with some other projects.